dev-docs/modules/userId.md

```diff
@@ -1366,6 +1370,9 @@ pbjs.setConfig({
     userSync: {
         userIds: [{
             name: "pubCommonId",
+            params: {
+                pixelUrl: "/wp-json/pubcid/v1/extend/"
```

Comment:
pixelUrl should be specified when the server plugin, so I would make this a no. 3 in the examples list and say something like 3) Publisher supports SharedID and first party domain cookie storage initiated by the server

Reply:
done